jupyterhub/auth.py

```diff
@@ -87,7 +94,28 @@ class LocalAuthenticator(Authenticator):
         should I try to create the system user?
         """
     )
-    
+
+    whitelist_group = Unicode(
+        config=True,
+        help="Automatically whitelist anyone in this group.",
+    )
+
+    def check_whitelist(self, username):
+        return (
+            super().check_whitelist(username) or
```

Comment:
> In the case where the user whitelist is empty, check_whitelist_group will never be called.

This is the behavior I intended, though that doesn't necessarily mean it's correct.

I'm trying to think if there's ever a case where it's useful to be able to say "allow people in this group, plus these random other users"...I could imagine a world where you had something like an "admin" group on a server, and you wanted to allow access to all the admins, plus a few exceptions who shouldn't have admin permissions, but should still be able to access the hub.